example/simple_example.go

```
@@ -0,0 +1,19 @@
+	"github.com/khulnasoft/khulnasoft-go/statsd"
+)
+
+func main() {
+	client, err := statsd.New("127.0.0.1:8125",
+		statsd.WithTags([]string{"env:prod", "service:myservice"}),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	client.Histogram("my.metrics", 21, []string{"tag1", "tag2:value"}, 1)
```

golangci-lint (1.62.2)

Error return value of `client.Histogram` is not checked

(errcheck)